docs/storage.yml

```diff
@@ -171,7 +157,144 @@ paths:
           content:
             application/octet-stream:
               example: these are some file contents
-
+  /api/v1/AuthToken:
+    post:
+      tags:
+        - archive
+      summary: Get a storage authentication token
+      operationId: AuthToken_Post
+      responses:
+        200:
+          description: A successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      type: string
+                    example: ["token1", "token2"]
+  /api/v1/WebRestoreSession:
```

Comment:
Looks like there's a `WebRestoreSession` api path in the `connected-server.yml` too (not sure why there's 2?) but that one will also need to have the `v1` added.

Reply:
Otherwise lgtm :)